website/content/docs/import/azurekv.mdx

```diff
@@ -0,0 +1,91 @@
+---
+layout: docs
+page_title: Azure secret import source
+description: The Azure KeyVault source imports secrets from Azure to Vault.
+---
+
+# Azure secret import source
+
+Use the Azure source to import secret data from Azure KeyVault into your Vault instance.To use dynamic
+credentials with Azure import, ensure the [Azure secrets engine](/vault/docs/secrets/azure) is already configured.
+
+<Note>
+All secrets including expired ones will imported from the provided Azure KeyVault URI
+</Note>
+
+## Argument reference
+
+Refer to the [HCL syntax](/vault/docs/import#hcl-syntax-1) for arguments common to all source types.
+
+## Additional arguments
+
+Required:
+
+- `key_vault_uri` `(string)` - The URI of the Azure KeyVault you want to import from.
```

Comment:
To follow the existing pattern...

```suggestion
- `key_vault_uri` `(string: <required>)` - The URI of the Azure KeyVault you want to import from.
```

[Example page](https://developer.hashicorp.com/vault/api-docs/system/audit-hash#parameters)